filters/ThirdParty/filter_214_ABPVNList/platforms.txt

```diff
@@ -0,0 +1,5 @@
+!#if (adguard_ext_android_cb)
+!#include https://github.com/abpvn/abpvn/blob/master/filter/abpvn_content_blocker.txt /exclude="../../exclusions.txt"
+!#else
```

Comment:
It's something new for me.
Is it a correct syntax?
I cannot find anything about `!#else` here - https://kb.adguard.com/en/general/how-to-create-your-own-ad-filters#hints_syntax

Reply:
@zloyden please check it locally. In the list authors slack it was just pseudo-code by me, don't take it too seriously:)

Reply:
I had tried to build this locally, but unfortunately, there was no success.
For now I have used our `platformsIncluded` and `platformsExcluded`.

Reply:
wow, new filter — isn't is too much?

I believe it can be solved with negation
```
! version for AdGuard Content Blocker only
!#if (adguard_ext_android_cb)
...
!#endif

! for all other AdGuard products
!#if (!adguard_ext_android_cb)
...
!#endif
```

Reply:
After our discussion, it works fine (checked local build). Thank you:)